### PR TITLE
Update Cloudflared tunnel directions

### PR DESCRIPTION
**Pull request by mrrcollins**
I've updated the directions for setting up the Cloudflared tunnels. I didn't know where to host the images, so do with them what you must. 😁

### CODE REVIEW

File: Reverse-Proxy-with-Cloudflare-Tunnel.md [hunk 1]
```diff
@@ -48,16 +48,24 @@ But trust me, once you learnt, you will remember how to config without this guid
...
+3. `Cloudflared` > `Next`
 
-   <img src="https://user-images.githubusercontent.com/1336778/160879200-642609d7-7264-41ea-8b16-b99f95e7f446.png" width="800" />
+![Cloudflared-Next.jpg](https://cdn.collinsoft.com/file/cdn-collinsoft/ryancollins-org/2024/2024-01-31-cloudflared-next/cloudflared-next-1024x.jpg)
 
-5. Go to your Uptime Kuma instance. 
+4. Type a `Tunnel name` such as `uptime-kuma` and save tunnel.
+5. Click the clipboard icon to copy the run command.
+
+![Copy.jpg](https://cdn.collinsoft.com/file/cdn-collinsoft/ryancollins-org/2024/2024-01-31-copy/copy-1024x.jpg)
```
Comment (CommanderStorm) [inline Reverse-Proxy-with-Cloudflare-Tunnel.md:62]: <details><summary>Details</summary>
<p>

image


</p>
</details> 

```suggestion
   <img src="https://github.com/louislam/uptime-kuma-wiki/assets/26258709/2853ee57-cc50-4ae6-9688-e102038d9f57" width="800" />
```

File: Reverse-Proxy-with-Cloudflare-Tunnel.md [hunk 2]
```diff
@@ -48,16 +48,24 @@ But trust me, once you learnt, you will remember how to config without this guid
...
+
+6. Paste that into a text editor so you can copy the token from the end of the command.
+
+`cloudflared.exe service install eyJhIjoiZDA4ZGNiMTUXXXXXXXXXXXXXXXXXXXXXXXXXXXXX`
+
```
Comment (CommanderStorm) [inline Reverse-Proxy-with-Cloudflare-Tunnel.md:67]: Plese remove the space below after merging. Github won't let me comment the closing tags for some reason ^^

```suggestion
   ```cmd
   cloudflared.exe service install eyJhIjoiZDA4ZGNiMTUXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
   `` `
```

File: Reverse-Proxy-with-Cloudflare-Tunnel.md [hunk 3]
```diff
@@ -48,16 +48,24 @@ But trust me, once you learnt, you will remember how to config without this guid
...
-4. Click the token to copy it.
+3. `Cloudflared` > `Next`
 
-   <img src="https://user-images.githubusercontent.com/1336778/160879200-642609d7-7264-41ea-8b16-b99f95e7f446.png" width="800" />
+![Cloudflared-Next.jpg](https://cdn.collinsoft.com/file/cdn-collinsoft/ryancollins-org/2024/2024-01-31-cloudflared-next/cloudflared-next-1024x.jpg)
 
-5. Go to your Uptime Kuma instance. 
+4. Type a `Tunnel name` such as `uptime-kuma` and save tunnel.
+5. Click the clipboard icon to copy the run command.
+
```
Comment (CommanderStorm) [inline Reverse-Proxy-with-Cloudflare-Tunnel.md:61]: This is one of the gochas of md:
If empty lines are added github assumes this is now a different pice of conent => the indentation is reverted

```suggestion
```

**Review by CommanderStorm — COMMENTED** [review]
I have changed the images from your CDN to ones hosted by GitHub (via the trick of embedding them in the comment and then using said image. If there is a better approach… 🤷🏻‍♂️)

**Comment by mrrcollins** [comment]
> I have changed the images from your CDN to ones hosted by GitHub (via the trick of embedding them in the comment and then using said image. If there is a better approach… 🤷🏻‍♂️)

I had forgotten about that trick! I'll remember that for next time. 😁

**Pull request merged**